platforms/web/vite.config.ts

```diff
@@ -72,8 +80,8 @@ export default ({ mode, command }: ConfigEnv): UserConfigExport => {
           theme_color: '#DD0000',
           orientation: 'any',
           background_color: '#000',
-          related_applications: [],
-          prefer_related_applications: false,
+          related_applications: env.APP_GOOGLE_RELATED_APPLICATION_ID ? [{ platform: 'play', id: env.APP_GOOGLE_RELATED_APPLICATION_ID, url: '' }] : [],
```

Comment:
I believe we can discard the (empty) URL.

According to the Google doc you mentioned the `url` property is not a requirement. Also in some other online examples, I don't see it. If we still want to provide the URL parameter maybe we can fill it with `https://play.google.com/store/apps/details?id=${env.APP_GOOGLE_RELATED_APPLICATION_ID}` an empty value is a bit weird.

Also, I find it a little bit more neat to also provide the itunes platform within `related_applications`, although this is not a requirement for the smart banner. It makes it more "complete".

To improve the readability I will also move this up in a `const`. You could move it to the buildTools.ts too but I leave this last decision up to you.

Reply:
I only added the empty string `url` because the typings of VitePWA expect it to be set.

Great idea to also add `itunes`, but that one does require a url, which cannot really be generated based on the bundle id. So that would imply another env var, basically for no practical purpose. What do you think?

Do agree about the `const`.

Reply:
> I only added the empty string url because the typings of VitePWA expect it to be set.

Then we could use my alternative approach with `https://play.google.com/store/apps/details?id=${env.APP_GOOGLE_RELATED_APPLICATION_ID}`

> Great idea to also add iTunes

Not important enough for now, then.

Reply:
Agreed, added it.
Also, I was wrong about the itunesId: not the bundle id, but the apple id is required, which I can now also use in the related_application. Please see my latest commit.

Reply:
Approved